qwik-graphql-tailwind/src/routes/auth/signin/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { component$ } from '@builder.io/qwik';
+import type { DocumentHead } from '@builder.io/qwik-city';
+import { SIGN_IN_URL } from '~/utils/constants';
+import * as styles from './signin.classNames';
+
+export default component$(() => {
+  return (
+    <section className={styles.section}>
+      <button
+        className={styles.button}
+        onClick$={() => {
+          window.location.href = SIGN_IN_URL;
```

Comment:
Using `window.location.href` feels hacky, could we use [redirect](https://qwik.builder.io/qwikcity/data/redirects/) instead?